opm/input/eclipse/EclipseState/Grid/PinchMode.hpp

```diff
@@ -20,22 +20,24 @@
 #ifndef OPM_PINCHMODE_HPP
 #define OPM_PINCHMODE_HPP
 
+#include <iosfwd>
 #include <string>
 
 namespace Opm {
     
-    namespace PinchMode {
-        enum ModeEnum {
-            ALL = 1,
-            TOPBOT = 2,
-            TOP = 3,
-            GAP = 4,
-            NOGAP = 5,
-        };
-
-        const std::string PinchMode2String(const ModeEnum enumValue);
-        ModeEnum PinchModeFromString(const std::string& stringValue);
-    }
+enum class PinchMode {
+    ALL = 1,
+    TOPBOT = 2,
+    TOP = 3,
+    GAP = 4,
+    NOGAP = 5,
+};
+
+const std::string PinchMode2String(const PinchMode enumValue);
```

Comment:
If we're touching this anyway, maybe we can update the declaration to return just a mutable `std::string` instead of a `const std::string` here?

Reply:
done.